misk/src/main/kotlin/misk/web/jetty/ThreadPoolQueueMetrics.kt

```diff
@@ -0,0 +1,20 @@
+package misk.web.jetty
+
+import misk.metrics.Metrics
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class ThreadPoolQueueMetrics @Inject internal constructor(
+  metrics: Metrics
+) {
+  private val queueLatencyHistogram = metrics.histogram(
```

Comment:
don't we have other jetty metrics? Ideally all `jetty_thread_pool_` metrics would be co-located.

Reply:
We have [ThreadPoolMetrics](https://github.com/cashapp/misk/blob/master/misk/src/main/kotlin/misk/web/jetty/ThreadPoolMetrics.kt#L9) but this depends on `QueuedThreadPool` which is created inside `MiskWebModule`. Hence, adding a dependency to `ThreadPoolMetrics` in `MiskWebModule` is not possible as it will be circular.